shared/common/ui/select/index.tsx

```diff
@@ -179,19 +189,41 @@ export function Select<T extends any>({
             className={cn(styles.tabDropdown, {
               [styles.tabDropdownOpen]: dropdownOpen,
               [styles.rightAlign]: !!rightAlign,
+              [styles.fullScreen]: !!fullScreen,
             })}
-            style={{maxHeight: maxHeight ?? maxHeight + "px"}}
+            style={isMobile ? {} : {maxHeight}}
             onClick={(e) => e.stopPropagation()}
           >
-            {searchable ? (
-              <input
-                ref={searchRef}
-                className={styles.searchInput}
-                placeholder="Search..."
-                value={searchFilter}
-                onChange={(e) => setSearchFilter(e.target.value)}
-              />
-            ) : null}
+            {isMobile && !!fullScreenTitle && (
+              <p className={styles.dropdownTitle}>{fullScreenTitle}</p>
+            )}
+            {isMobile && !!fullScreen && (
+              <button
+                className={styles.closeDropdown}
+                onClick={() => setDropdownOpen(false)}
+              >
+                <CrossIcon />
+              </button>
+            )}
+            {!!searchable &&
+              (fullScreen ? (
```

Comment:
I think this should be `fullScreen && isMobile`

Reply:
Actually, there's a few places where you use just `fullScreen` that I think should also be `fullScreen && isMobile`. Maybe change the `const isMobile = useIsMobile();` to something like `const isFullscreenMobile = useIsMobile() && fullscreen;`?